intro-HPC/ch_connecting.tex

```diff
@@ -504,6 +504,13 @@ \section{First Time connection to the HPC infrastucture}
 user interface. Usually a locale identifier consists of at least a language
 identifier and a region identifier.
 
+\ifgent
+\strog{Note:} If you try to set on a UGent HPC cluster a non-supported locale,
```

Comment:
remove the `on a UGent HPC cluster`, this is a `\ifgent` block

Reply:
Done.